pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -113,6 +118,11 @@ func NewNetworkPolicyController(antreaClientGetter agent.AntreaClientProvider,
 			if !ok {
 				return fmt.Errorf("cannot convert to *v1beta1.NetworkPolicy: %v", obj)
 			}
+			if !c.antreaPolicyEnabled && policy.SourceRef.Type != v1beta1.K8sNetworkPolicy {
+				klog.Infof("Ignore Antrea NetworkPolicy %s since AntreaProxy feature gate is not enabled",
```

Comment:
```suggestion
				klog.Infof("Ignore Antrea NetworkPolicy %s since AntreaPolicy feature gate is not enabled",
```

here and in below log messages

Reply:
Thanks. Fixed. Do not know why I mistook Policy to Proxy in multiple places..